.github/workflows/validate-nightly-pypi-wheel-binary-size.yml

```diff
@@ -0,0 +1,26 @@
+name: Validate Nightly PyPI Wheel Binary Size
+on:
+  pull_request:
+    paths:
+      - .github/workflows/validate-nightly-pypi-wheel-binary-size.yml
+  workflow_dispatch:
+  schedule:
+    # At 2:30 pm UTC (7:30 am PDT)
+    - cron: "30 14 * * *"
+
+jobs:
+  nightly-pypi-binary-size-validation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          repository: pytorch/test-infra
```

Comment:
If we're checking out `pytorch/test-infra` then why don't we just have this workflow there?

Reply:
Unfortunately `test-infra` currently doesn't support alerts (see [action run](https://github.com/pytorch/test-infra/actions/runs/4137871658/jobs/7153578233)). Adding workflow in builder repo is a temporary solution, until we [enable alerts](https://github.com/pytorch/test-infra/blob/main/torchci/docs/architecture.md#adding-a-new-repo-to-torchci) in `test-infra`.